packages/grafana-data/src/types/azure.ts

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
This looks very azure specific, so it should probably not live in grafana/data. If it's only azure monitor that uses it it could live in that datasource plugin, but if we want to use it in external datasource plugins such as ADX, we should put it in an npm package.

Reply:
Yes, removed it for now. I made the `cloud` property just a `string` to not complicate things.
Later will will decide where to put strongly-typed definitions.